srcpkgs/sequoia/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'sequoia'
+pkgname=sequoia
+version=0.16.0
+revision="1"
+wrksrc="${pkgname}-v${version}"
+build_style="gnu-makefile"
+build_helper="rust"
+make_install_args="PYTHON=disable"
```

Comment:
Why disable the python stuff?

Reply:
It disables building python ffi bindings. I think it should go to a different package.